catalog/app/components/FileEditor/QuiltConfigEditor/WorkflowsToolbar.tsx

```diff
@@ -0,0 +1,320 @@
+                size="small"
+              />
+              <RF.Field
+                component={Form.Field}
+                disabled={submitting}
+                errors={emptyObject}
+                fullWidth
+                helperText={<Code>package_handle</Code>}
+                label="Regular expression to validate package handle"
+                margin="normal"
+                name="handle_pattern"
+                placeholder="e.g. ^foo/bar$"
+                size="small"
+              />
+              <RF.Field
+                bucket={bucket}
+                component={SchemaField}
+                disabled={submitting}
+                errors={emptyObject}
+                fullWidth
+                helperText={<Code>metadata_schema</Code>}
+                label="Metadata JSON Schema name"
+                margin="normal"
+                name="metadata_schema"
+                placeholder="e.g. mySchema1"
+                size="small"
+                validate={validateSchemaName}
+              />
+              <RF.Field
+                bucket={bucket}
+                component={SchemaField}
+                disabled={submitting}
+                errors={emptyObject}
+                fullWidth
+                helperText={<Code>entries_schema</Code>}
+                label="Entries JSON Schema name"
+                margin="normal"
+                name="entries_schema"
+                placeholder="e.g. mySchema1"
+                size="small"
+                validate={validateSchemaName}
+              />
+              <RF.Field
+                FormControlProps={FormControlProps}
+                component={Form.Checkbox}
+                label="Is message required"
+                name="is_message_required"
+                type="checkbox"
+              />
+            </M.DialogContent>
+            <M.DialogActions>
+              {(!!error || !!submitError) && <FormError error={error || submitError} />}
+              <M.Button onClick={onClose} disabled={submitting}>
+                Cancel
+              </M.Button>
+              <M.Button
+                type="submit"
+                onClick={handleSubmit}
+                variant="contained"
+                color="primary"
+                disabled={submitting || (submitFailed && hasValidationErrors)}
+              >
+                Save
+              </M.Button>
+            </M.DialogActions>
+          </>
+        )}
+      </RF.Form>
+    </M.Dialog>
+  )
+}
+
+interface ToolbarProps {
+  bucket: string
+  onChange: (value: WorkflowYaml) => void
+}
+
+function Toolbar({ bucket, onChange }: ToolbarProps) {
+  const [open, setOpen] = React.useState(false)
+  const handleSubmit = React.useCallback(
+    (value: FormValues) => {
+      onChange(value)
+      setOpen(false)
+    },
+    [onChange],
+  )
+
+  return (
+    <>
+      <Button onClick={() => setOpen(true)} />
+      <Popup
+        bucket={bucket || 'fiskus-sandbox-dev'}
```

Comment:
are you hardcoding `fiskus-sandbox-dev` bucket intentionally?

Reply:
Unintentional, while I was debugging in `/storybook` where is no `bucket`